sdk/src/main/scala/io/horizen/account/state/events/AddCrossChainMessage.scala

```diff
@@ -0,0 +1,34 @@
+package io.horizen.account.state.events
+
+import io.horizen.account.state.events.annotation.{Indexed, Parameter}
+import io.horizen.evm.Address
+import org.web3j.abi.datatypes.DynamicBytes
+import org.web3j.abi.datatypes.generated.Uint32
+
+import scala.annotation.meta.getter
+
+case class AddCrossChainMessage(@(Parameter@getter)(1) @(Indexed@getter) sender: Address,
+                                @(Parameter@getter)(2) @(Indexed@getter) messageType: Uint32,
+                                @(Parameter@getter)(3) receiverSidechain: DynamicBytes,
```

Comment:
```suggestion
                                @(Parameter@getter)(3) receiverSidechain: Bytes32,
```
use as much precise types as it is possible